### DISCUSSION TITLE
How can you use a Map in HScript?

### DISCUSSION THREAD

**Discussion by TechnikTil** [FunkinCrew/Funkin#2941] [Q&A] [answered]
How can you use a Map in HScript, or any abstracts in that case?

**Comment by riconuts** [comment, marked as answer]
In the case of Map, you should use StringMap, ObjectMap, EnumValueMap or IntMap, as that's what the Map abstract turns into in compiled Haxe code. For the rest there's just not always a clear alternative and you'll have to find your way around them as abstracts don't exist during runtime.

**Reply by TechnikTil** [reply]
Thanks!
